### PR TITLE
Feature/apadrinhamento

### PR DESCRIPTION
**Pull request by dutraneto**
Create "apadrinhamento" page with CTAs to lead to 'apadrinhar' page and 'ser-apadrinhado' page

### CODE REVIEW

File: components/Padrinho/PadrinhoContent.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link'
```
Comment (marceloglacial) [inline components/Padrinho/PadrinhoContent.js]: Rename the file to Pedrinho.js (each component needs a root file with the same folder name (index).

File: pages/apadrinhamento.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import Button from 'components/Button/Button'
+import { getData } from 'functions/getData';
+
+const Apadrinhamento = (props) => {
+  const { headerMenu } = props
+
+  return (
+    <Layout pageTitle={`Torne-se um Padrinho`} {...props}>
+      <PadrinhoContent title={`Apadrinhe um recém-chegado ao Canada!`} />
+      <section className={`text-center`}>
```
Comment (marceloglacial) [inline pages/apadrinhamento.js]: Move this section to the component itself. You can split in small components there, but keep it on the same context.

File: components/Padrinho/PadrinhoContent.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        </div>
+        <div>
+          <p>Pensando em facilitar o encontro entre essas pessoas na comunidade, a BRINCA criou o projeto de apadrinhamento. Através de um cadastro simples, selecionamos os perfis que mais se adequam e o contato entre padrinho e apadrinhado é feito. A partir daí os dois lados poderão combinar a melhor forma de comunicação e periodicidade dos encontros. Não há limites!</p>
+        </div>
+        <div>
+          <p>Queremos criar uma comunidade cada vez mais forte e unida, e acreditamos que esse link possa criar laços fortes entre as pessoas. Aquele que foi ajudado pode ser o que ajuda no futuro, e assim todos ganham.</p>
+        </div>
+        <div>
+          <p>Para participar do nosso projeto é só clicar no link abaixo e fazer o seu cadastro, ou envie um e-mail para <Link href="mailto:padrinho@brinca.ca?subject=Cadastro - Gostaria de ser Padrinho/Apadrinhado">padrinho@brinca.ca</Link>. Este é um trabalho 100% voluntário e coordenado também por voluntários, portanto, você não precisa pagar nada.</p>
```
Comment (marceloglacial) [inline components/Padrinho/PadrinhoContent.js]: Link component is related to internal pages and route. In this case, use `<a href>` only

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: components/Padrinho/PadrinhoContent.js (hunk 1 above)
Comment (dutraneto) [reply components/Padrinho/PadrinhoContent.js]: Done

File: pages/apadrinhamento.js (hunk 2 above)
Comment (dutraneto) [reply pages/apadrinhamento.js]: Is that ok if I just get rid of the section of buttons and keep everything within its component? Just want to make it cleaner

File: components/Padrinho/PadrinhoContent.js (hunk 3 above)
Comment (dutraneto) [reply components/Padrinho/PadrinhoContent.js]: Totally my bad

File: pages/apadrinhamento.js (hunk 2 above)
Comment (marceloglacial) [reply pages/apadrinhamento.js]: yes

File: components/Padrinho/PadrinhoContent.js (hunk 3 above)
Comment (marceloglacial) [reply components/Padrinho/PadrinhoContent.js]: no worries at all

File: pages/apadrinhamento.js (hunk 2 above)
Comment (dutraneto) [reply pages/apadrinhamento.js]: done

File: components/Padrinho/PadrinhoContent.js (hunk 3 above)
Comment (dutraneto) [reply components/Padrinho/PadrinhoContent.js]: done

**Review by marceloglacial — APPROVED** [review]
Good job

**Pull request merged**